packages/protocol/test/rrp/AirnodeRrp.sol.js

```diff
@@ -442,32 +457,59 @@ describe('fulfill', function () {
                 ]
               )
             );
-            // Attempt to fulfill the request
             const sponsorWallet = utils
               .deriveSponsorWallet(airnodeMnemonic, roles.sponsor.address)
               .connect(hre.ethers.provider);
-            const fulfillStatusCode = 0;
             const fulfillData = hre.ethers.utils.keccak256(
               hre.ethers.utils.solidityPack(['uint256', 'string'], ['123456', 'hello'])
             );
-            // Reverts because the target function always reverts
+            // Make static call to check fulfillment result
+            // Returns false because the target function always reverts
+            const staticCallResult = await airnodeRrp
+              .connect(sponsorWallet)
+              .callStatic.fulfill(
+                requestId,
+                airnodeAddress,
+                fulfillData,
+                rrpRequester.address,
+                rrpRequester.interface.getSighash('fulfillAlwaysReverts'),
+                { gasLimit: 500000 }
+              );
+            expect(staticCallResult.callSuccess).to.equal(false);
+            expect(utils.decodeRevertString(staticCallResult.callData)).to.equal('Always reverts');
```

Comment:
This is how we retrieve the revert string to use in `errorMessage` for the `fail()` call